Snakefile

```diff
@@ -420,92 +637,1344 @@ rule atlas_experiment_summary:
+        getagg=rules.summarize_transcripts.output
+    params:
+        organism=get_organism(),
+        transcripts="{accession}-transcripts-{metric}.tsv.undecorated.aggregated",
+        decotranscripts="{accession}-transcripts-{metric}.tsv"
+    output:
+        temp("logs/{accession}-transcripts-{metric}.tsv.done")
+    shell:
+        """
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
+        source {workflow.basedir}/bin/reprocessing_routines.sh
+
+        # transcripts undecorated-aggregated rule done
+        echo {input.getagg}
+
+        if [ -s {params.transcripts} ] ; then
+
+            geneNameFile=$( get_geneNameFile_given_organism {params.organism}  )
+            transcriptFile=$( get_transcriptFile_given_organism {params.organism}  )
+
+            echo $geneNameFile
+            echo $transcriptFile
+
+            test -s "$geneNameFile" || (  >&2 echo "$0 gene name file not found: $geneNameFile" ; exit 1 )
+            test -s "$transcriptFile" || (  >&2 echo "$0 transcript to gene mapping file not found: $transcriptFile" ; exit 1 )
+
+            decoratedFile={params.decotranscripts} 
+
+            # Ammonite REPL & Script-Runner
+            #export JAVA_OPTS="-Xmx3000M"
+            amm -s {workflow.basedir}/bin/decorateFile.sc \
+                --geneIdFile "$transcriptFile" \
+                --geneNameFile "$geneNameFile" \
+                --source {params.transcripts} \
+                | awk 'NR == 1; NR > 1 {{print $0 | "sort -n"}}' \
+                > $decoratedFile.swp
+
+            decoratedFileLength=$(wc -l "$decoratedFile.swp" | cut -f 1 -d ' ' )
+            if [ -s "$decoratedFile.swp" ] && [ "$decoratedFileLength" -gt 1 ]; then
+                mv $decoratedFile.swp $decoratedFile
+            else
+                echo "ERROR: decorate_transcript_baseline_file for {wildcards.accession} and {wildcards.metric}"
+	            exit 1
+            fi
+        else
+            echo "File {params.transcripts} not found. Decorate transcript for baseline rna-seq summary not performed "
+        fi
+        touch {output}
+        """
+
+
+rule create_tracks_symlinks:
+    """
+    Create bedgraph tracks symlinks during reprocessing (only for tpms).
+    """
+    log: "logs/{accession}.{assay_id}.create_tracks_symlinks_{metric}.log"
+    input:
+        bedGraph="{accession}.{assay_id}.genes.expressions_{metric}.bedGraph"
+    params:
+        output="{accession}.{assay_id}.genes.expressions.bedGraph"
+    output:
+        temp("logs/{accession}.{assay_id}.create_tracks_symlinks_{metric}.done")
+    shell:
+        """
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
+
+        # remove if any old bedgraph files
+        if [ -s {params.output} ] ; then
+            rm {params.output} 
+        fi
+
+        ln -s {input.bedGraph} {params.output}
+        touch {output}
```

Comment:
@pcm32 I did that to avoid to avoid the uncertainty error in Snakemake produced because the output file has no wildcard `{metric}` on it. Hence the required output was: 
```
        outputs.extend(expand(f"{config['accession']}"+".{a_id}.create_tracks_symlinks_{metric}.done",
                            a_id=get_assay_ids(),
                            metric=["tpms"]))
```
Now, I have removed this wildcard, as only TPMs are required, 68be696